### DISCUSSION TITLE
How to create a window which is in maximized mode by default.

### DISCUSSION THREAD

**Discussion by Aryan-29** [fltk-rs/fltk-rs#571] [Q&A] [answered]
FLTK-Rust is a great GUI creation library... but I want to know how to create a window that is in maximized mode(not full screen as in OS like windows the taskbar also gets disappeared). I am able to do it manually by clicking the button on the top right but I want to implement this in the code itself.

**Comment by MoAlyousef** [comment, marked as answer]
Hi
Thanks.
FLTK doens't offer that functionality out of the box since it's something normally provided by the window manager. For example on windows, you can tell windows which apps to run maximized by default like here:
https://www.computerhope.com/tips/tip201.htm

In FLTK you can get the screen size using app::screen_size(). Then set the size of window to be a smaller fraction of that. You might have to fiddle a bit to find the best ratio, and then it might look different on other window managers or operating systems.
